### ISSUE TITLE
Finetuning Alex-Net for Semantic Segmentation

### ISSUE THREAD

**Issue by seragENTp** [vlfeat/matconvnet-fcn#14]
I'm trying to reproduce FCN paper results using Alex-Net (imagenet-matconvnet-alex.mat) which 
doesn't contain  Local response normalization layer as in Alex-Net (imagene-caffe-ref.mat).
This the results i obtained using :
minibatch size = 1 ;
learning rate = 1e-5 ;
after 100 epoch i got meanIU = 25 on validation dataset (1111 images), so what i should do next !

1-train for more epochs ?
2- decrease the learning rate  and train from the start  ?
3- add local response normalization layer like the paper ? 

[net-train.pdf](https://github.com/vlfeat/matconvnet-fcn/files/91654/net-train.pdf)
